docs/jupyter-lite.json

```diff
@@ -5,7 +5,7 @@
     "appUrl": "./lab",
     "litePluginSettings": {
       "@jupyterlite/pyodide-kernel-extension:kernel": {
-        "pyodideUrl": "/static/pyodide/pyodide.js"
+        "pyodideUrl": "https://cdn.jsdelivr.net/pyodide/dev/full/pyodide.js"
```

Comment:
For now we must use the `dev` version (rolling release pointing to `pyodide@main`) because it is the only distro over a public CDN which already includes https://github.com/pyodide/pyodide/pull/5183  and https://github.com/pyodide/pyodide/pull/5184

Reply:
Sounds good